frontend/src/metabase/embedding-sdk/theme/default-component-theme.ts

```diff
@@ -1,10 +1,7 @@
 import { merge } from "icepick";
 
 import { OVERLAY_Z_INDEX } from "metabase/css/core/overlays/constants";
-import {
-  EMBEDDING_SDK_FULL_PAGE_PORTAL_ROOT_ELEMENT_ID,
-  EMBEDDING_SDK_PORTAL_ROOT_ELEMENT_ID,
-} from "metabase/embedding-sdk/config";
+import { EMBEDDING_SDK_PORTAL_ROOT_ELEMENT_ID } from "metabase/embedding-sdk/config";
```

Comment:
change from https://github.com/metabase/metabase/pull/60431, I'll remove it once that PR lands.